vega-lite-schema.json

```diff
@@ -1341,7 +1341,7 @@
             ],
             "type": "object"
         },
-        "GenericFacetSpec<AnyMark, EncodingWithFacet>": {
+        "GenericFacetSpec<GenericUnitSpec<AnyMark, EncodingWithFacet>>": {
```

Comment:
Then I guess we can merge this, assuming that we will turn that nice name flag on soon?